src/Microsoft.Extensions.Caching.SqlConfig.Tools/Program.cs

```diff
@@ -103,7 +103,7 @@ private void CreateTableAndIndexes()
                     _logger.LogWarning(
                         $"Table with schema '{_schemaName}' and name '{_tableName}' already exists. " +
                         "Provide a different table name and try again.");
-                    return;
+                    return 1;
                 }
 
                 reader.Dispose();
```

Comment:
I know it is not related to your change but I just noticed that this will not be called if the query returns non-empty reader